mlos_bench/mlos_bench/config/schemas/tunables/tunable-params-schema.json

```
@@ -73,6 +73,12 @@
                     "minItems": 2,
                     "maxItems": 2
                 },
+                "quantization": {
+                    "type": "number"
```

```suggestion
                    "description": "The number of buckets to divide the range into.  See Also: https://automl.github.io/ConfigSpace/main/api/hyperparameters.html#module-ConfigSpace.api.types.float"
                    "type": "number"
```

Add a descriptive text that can appear during editing with supported IDEs.

Similar for the "log" parameter types.

---

BTW, I think it should be possible to consolidate the "integer" and "float" param types in the schema a bit to reduce a little duplication.  I'll send a PR along for that.

---

https://github.com/motus/MLOS/pull/8

---

merged, thank you!